types/spdx-satisfies/index.d.ts

```diff
@@ -1,2 +1,2 @@
-declare function spdxSatisfies(first: string, second: string): boolean;
+declare function spdxSatisfies(spdxExpression: string, arrayOfLicenses: string[]): boolean;
```

Comment:
Since the function doesn't modify the array, it can be marked `readonly`:

```suggestion
declare function spdxSatisfies(spdxExpression: string, arrayOfLicenses: readonly string[]): boolean;
```